bundles/test/org.eclipse.smarthome.test/src/test/java/org/eclipse/smarthome/test/java/JavaTestTest.java

```diff
@@ -52,15 +52,15 @@ public void waitForAssertShouldRunAfterLastCall_whenAssertionSucceeds() {
     public void waitForAssertShouldRunAfterLastCall_whenAssertionFails() {
         Runnable allwaysFailAssertion = mock(Runnable.class);
         doAnswer(invocation -> {
-            throw new NullPointerException();
+            throw new AssertionError();
```

Comment:
@htreu 
After I added "src/main/java/" and "src/test/java/" to the source property in build.properties file for package "org.eclipse.smarthome.test" the build failed with the error @maggu2810 pointed out for "org.eclipse.smarthome.test.java.JavaTestTest" file.

Found out that one of the tests in JavaTestTest - waitForAssertShouldRunAfterLastCall_whenAssertionFails  was throwing and catching NullPointerException instead of AssertionError which should be the correct one according the Javadoc.

After fixing it, the test passed and I expect the build to succeed.